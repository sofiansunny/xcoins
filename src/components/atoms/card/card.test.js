import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import CardView from '.';
describe('SomeComponent component', () => {
  it('Shallow rendering', () => {
    const props = {
      children: null,
      style: {},
    };
    const wrapper = shallow(<CardView {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})
