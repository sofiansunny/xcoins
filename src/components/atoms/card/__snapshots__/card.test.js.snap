// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomeComponent component Shallow rendering 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "borderColor": "#e4f1f7",
        "borderRadius": 7.857142857142857,
        "borderWidth": 2.142857142857143,
        "justifyContent": "center",
        "paddingHorizontal": 42.857142857142854,
        "paddingVertical": 21.428571428571427,
      },
      Object {},
    ]
  }
/>
`;
